README_v_0_2_x.md

```diff
@@ -64,17 +77,79 @@ or
 
-## Testing `pyani`
+## Installing from the repository/source code
+
+If you wish to install `pyani` v0.2 from source code, you will need to download this code from GitHub either directly, or by cloning the repository.
+
+Both methods will create a directory called `pyani` which contains the source code for v0.2.
+
+### IMPORTANT NOTICE
+
+**THE DEFAULT REPOSITORY ON `GITHUB` IS `PYANI` v0.3+, *NOT* v0.2. THE INSTRUCTIONS ON THE MAIN REPOSITORY `README.md` DO NOT APPLY TO v0.2**
+
+Please note that v0.2 is a **MAINTENANCE-ONLY RELEASE**. This means that bugs and security issues will be fixed, but no new features will be added. All new feature development now takes place under v0.3 (the `master` branch on the repository).
+
+### Direct download
+
+Download the source as `.zip` or `.tar.gz` from [https://github.com/widdowquinn/pyani/releases/tag/v0.2.10](https://github.com/widdowquinn/pyani/releases/tag/v0.2.10)
+
+Extract the archived file using the appropriate commands (or by double-clicking in your file explorer/finder):
 
-`pyani` includes tests that can be run with `nosetest` (including coverage measurement using `coverage.py`) with the following command, executed from the repository root directory:
+```bash
+unzip v0.2.10.zip
+```
+
+or
 
 ```bash
-nosetests --cover-erase --cover-package=pyani --cover-html --with-coverage
+tar -zxvf v0.2.10.tar.gz
 ```
 
-Coverage output will be placed (by default) in the `cover` subdirectory, and can be loaded into the web browser.
+### Clone the repository using `git`
+
+Download the current `pyani` repository with `git clone`, and change branch to `version_0_2`:
+
+```bash
+git clone git@github.com:widdowquinn/pyani.git
+git checkout version_0_2
+```
+
+### Installation
+
+Change directory to `pyani`, and use the `python setup.py` setuptools command, to install the package and scripts. This will **not** install the third-party tools `BLAST`/`BLAST+` and `mummer`.
+
+```bash
+cd pyani
+python setup.py build
+python setup.py install
+```
+
+### Third-party tools
+
+Three alignment packages are required, to use all of `pyani`'s methods: `mummer`, `BLAST+`, and legacy `BLAST`.
+
+The simplest route to obtaining these tools is to use `conda`/`bioconda`:
+
+```bash
+conda install mummer blast legacy-blast -y
+```
+
+But they can also be installed by following instructions from the tools' own websites.
+
+- **BLAST+** (for `anib`) [ftp://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/LATEST/](ftp://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/LATEST/)
+- **legacy BLAST** (for `aniblastall`) [ftp://ftp.ncbi.nlm.nih.gov/blast/executables/release/LATEST/](ftp://ftp.ncbi.nlm.nih.gov/blast/executables/release/LATEST/)
+- **MUMmer** (for `anim`) [http://mummer.sourceforge.net/](ftp://ftp.ncbi.nlm.nih.gov/blast/executables/release/LATEST/)
```

Comment:
These links also don't look like they're working in the preview.

Reply:
Yes. I think that's another GitHub Markdown rendering choice.